src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs

```diff
@@ -47,6 +48,27 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 return TaskCache.CompletedTask;
             }
 
+            // The following check causes the ComplexTypeModelBinder to NOT participate in binding structs as 
+            // reflection does not provide information about the implicit parameterless constructor for a struct.
+            // This binder would eventually fail to construct an instance of the struct as the Linq's NewExpression
+            // compile fails to construct it.
+            var modelTypeInfo = bindingContext.ModelType.GetTypeInfo();
+            if (bindingContext.Model == null
+                && (modelTypeInfo.IsAbstract
+                || modelTypeInfo.GetConstructor(Type.EmptyTypes) == null))
```

Comment:
Move operators to the ends of the lines. This isn't even consistent with the rest of the file e.g. [lines 97-99](https://github.com/aspnet/Mvc/blob/2c796b93c3885a626d64da5773f4c5a761cba7f0/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs#L97-L99).